scalactic/src/main/scala/org/scalactic/Catcher.scala

```diff
@@ -98,7 +98,7 @@ val InternalServerError =
  *
  * @param partial the partial function that is used by this extractor to determine matches
  */
-class Catcher(partial: PartialFunction[Throwable, Boolean]) {
+class Catcher(partial: PartialFunction[Throwable, Boolean] | Null) {
```

Comment:
**Nullable parameter**
Parameter that can be null and should therefore have a `| Null` type.